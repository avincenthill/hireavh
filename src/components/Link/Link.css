h2 {
  margin-top: 0;
  margin-bottom: 0;
}

a {
  display: flex;
  align-items: center;
}