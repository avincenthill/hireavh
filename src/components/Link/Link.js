import React from "react";
import styleconfig from "../../styles/styleconfig";
import { IconContext } from "react-icons";
import { GoChevronRight } from "react-icons/go";
import "./Link.css";

class Link extends React.Component {
  render() {
    return (
      <div>
        <h2>
          <a href={this.props.path}>
            <IconContext.Provider value={{ style: styleconfig.icons.link }}>
              <GoChevronRight></GoChevronRight>
            </IconContext.Provider>
            <span>{this.props.title}</span>
          </a>
        </h2>
      </div>
    );
  }
}

export default Link;
