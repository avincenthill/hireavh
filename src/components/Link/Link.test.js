import React from "react";
// import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Link from "./Link";

describe("Link", () => {
  describe("snapshot", () => {
    it("matches the last saved snapshot", () => {
      const tree = renderer.create(<Link />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
