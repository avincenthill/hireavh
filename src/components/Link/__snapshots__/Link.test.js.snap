// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link snapshot matches the last saved snapshot 1`] = `
<div>
  <h2
    className="link"
  >
    <a
      className="link-a"
    >
      <span />
    </a>
  </h2>
</div>
`;
